src/Haskell/Ide/Engine/Version.hs

```diff
@@ -1,93 +1,26 @@
-{-# LANGUAGE CPP                   #-}
-{-# LANGUAGE DataKinds             #-}
-{-# LANGUAGE GADTs                 #-}
-{-# LANGUAGE OverloadedStrings     #-}
-{-# LANGUAGE PartialTypeSignatures #-}
-{-# LANGUAGE TemplateHaskell       #-}
-module Haskell.Ide.Engine.Plugin.Base where
+{-# LANGUAGE CPP, TemplateHaskell, OverloadedStrings #-}
```

Comment:
Why are you collapsing the pragmas?

Reply:
I am not sure. I think this came in when shuffling parts about from the old PR. Will fix